rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMListenerProxy.java

```diff
@@ -127,7 +127,7 @@ public boolean init(String url, Credentials credentials) throws RMException, Key
     public BooleanWrapper disconnect() {
         try {
             target.getMonitoring().removeRMEventListener();
-        } catch (RMException ignored) {
+        } catch (Exception ignored) {
```

Comment:
Can we be a bit more selective on the exception to be intercepted ?

Reply:
not in the context of removal, close, clean, we want to be as large as possible